kvmagent/kvmagent/test/http/conftest.py

```
@@ -0,0 +1,640 @@
+        )
+        return result.returncode, result.stdout, result.stderr
+    return _run
+
+
+def _deploy_remote_file(host, password, content, remote_path):
+    """Write content to remote host via temp file + SCP."""
+    with tempfile.NamedTemporaryFile(mode='w', suffix='.py', delete=False) as f:
+        f.write(content)
+        tmp = f.name
+    try:
+        subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)